pyproj/_geod.pyx

```diff
@@ -12,6 +15,43 @@ geodesic_version_str = (
     f"{GEODESIC_VERSION_MAJOR}.{GEODESIC_VERSION_MINOR}.{GEODESIC_VERSION_PATCH}"
 )
 
+GeodIntermediateReturn = namedtuple(
+    "GeodIntermediateReturn", ["npts", "del_s", "dist", "lons", "lats", "azis"]
+)
+
+cdef int GEOD_INTER_FLAG_DEFAULT = 0x0
+
+cdef int GEOD_INTER_FLAG_NPTS_MASK = 0xF
+cdef int GEOD_INTER_FLAG_NPTS_ROUND = 0x0
+cdef int GEOD_INTER_FLAG_NPTS_CEIL = 0x1
+cdef int GEOD_INTER_FLAG_NPTS_TRUNC = 0x2
+
+cdef int GEOD_INTER_FLAG_DEL_S_MASK = 0xF0
+cdef int GEOD_INTER_FLAG_DEL_S_RECALC = 0x00
+cdef int GEOD_INTER_FLAG_DEL_S_NO_RECALC = 0x10
+
+cdef int GEOD_INTER_FLAG_AZIS_MASK = 0xF00
+cdef int GEOD_INTER_FLAG_AZIS_DISCARD = 0x000
+cdef int GEOD_INTER_FLAG_AZIS_KEEP = 0x100
+
+
+class GeodIntemediateFlag(IntFlag):
```

Comment:
Mind moving this to `pyproj.enums` and adding a reference to it in `docs/api/enums.rst`?

Reply:
done